sites.json

```diff
@@ -5757,6 +5757,16 @@
     },
 
     {
+        "name": "Unity ID",
+        "url": "https://support.unity3d.com/hc/en-us/articles/209059926",
+        "difficulty": "hard",
```

Comment:
If it is not possible for an account to be deleted (only disabled), this should be `impossible` rather than `hard`.